skaffold.yaml

```diff
@@ -41,35 +41,56 @@ deploy:
         version: 11.6.2
         useHelmSecrets: true
         valuesFiles:
-          - helm/data/secrets/{{ .ENVIRONMENT }}/postgresql.yaml
-          - helm/data/values/{{ .ENVIRONMENT }}/postgresql.yaml
+          - helm/data/secrets/{{ .CLASSIFICATION }}/postgresql.yaml
+          - helm/data/values/{{ .CLASSIFICATION }}/postgresql.yaml
 
       - name: im-manager-{{ .ENVIRONMENT }}
         namespace: instance-manager-{{ .ENVIRONMENT }}
         createNamespace: true
         chartPath: helm/chart
         setValueTemplates:
-          image.repository: dhis2/im-manager
+          image:
+            repository: dhis2/im-manager
+          serviceAccount:
+            annotations:
+              eks\\.amazonaws\\.com\\/role-arn: arn:aws:iam::767224633206:role/instance-cluster-test-im-manager-{{ .ENVIRONMENT }}
```

Comment:
The key here needs to be quoted, otherwise I'm getting the following error when running `skaffold run/dev`

```
Error: UPGRADE FAILED: YAML parse error on im-manager/templates/serviceaccount.yaml: error unmarshaling JSON: while decoding JSON: json: cannot unmarshal object into Go struct field .metadata.annotations of type string
```